README.md

```diff
@@ -148,7 +148,11 @@ rustup target add riscv32i-unknown-none-elf
 
 ### Rust with Podman or Docker
 
-Alternatively you might build the project in the container where image already contains pre-installed Rust and ESP-IDF.
+Alternatively you might build the project in the container where the image already contains pre-installed Rust and ESP-IDF.
+
+There are two different images published to Dockerhub: 
+- [idf-rust](https://hub.docker.com/r/espressif/idf-rust) - contains only the toolchain.
+- [idf-rust-examples](https://hub.docker.com/r/espressif/idf-rust-examples) - includes example code from https://github.com/espressif/rust-esp32-example and is about 1GB larger in size. 
```

Comment:
```suggestion
- [idf-rust-examples](https://hub.docker.com/r/espressif/idf-rust-examples) - includes examples code from [rust-esp32-example](https://github.com/espressif/rust-esp32-example) and the second one is [rust-esp32-std-demo](https://github.com/ivmarkov/rust-esp32-std-demo) and is about 10GB larger in size. 
```
`idf-rust-examples` consists of 2 examples, first is [rust-esp32-example](https://github.com/espressif/rust-esp32-example) and the second one is [rust-esp32-std-demo](https://github.com/ivmarkov/rust-esp32-std-demo). Image size is than about 10GB.

Reply:
Ah, good point on the two examples.
Re: size -> I was referring to the size according to Dockerhub, where the image for `idf-rust-examples` is 3.4GB compared to 2.5GB for `idf-rust`. Should we just leave that part out?

Reply:
I see, I was referring to the downloaded size. I think we can leave that part out :)